_schema.graphql

```diff
@@ -9032,6 +9032,9 @@ type Partner implements Node {
 
   # The gallery partner's web address
   website: String
+
+  # Indicates the partner is a trusted seller on Artsy
+  commerceVerified: Boolean
```

Comment:
Is there any reason _not_ to name this `trustedSeller`?

Reply:
We had a little bit of discussion in the [gravity PR](https://github.com/artsy/gravity/pull/12609#discussion_r332211884) about the name of the flag and it just got carried over to here. You mean renaming it to `trustedSeller` only in MP or both here an gravity/kinetic. Happy to do either if it makes it more clear.

Reply:
I think only in MP. The name doesn't have to match underlying systems IMO, but generally I'd like to keep the MP as self evident as possible. Maybe @alloy has opinions?

Reply:
I agree. Maybe it’s my naive English, but “commerce verified” doesn’t really mean anything to me whereas `isTrustedSeller` or `isVerifiedSeller` do. While having docs is always good, the greatest of names don’t _need_ docs.

Reply:
(Docs could expand on eg what criteria is considered for a partner to have that status.)

Reply:
Yes I also agree. Now that we have the opportunity to also rename it in gravity would you suggest doing that to avoid confusion? May be `verified_seller` in gravity and `isVerifiedSeller` in MP to follow the variable naming conventions in both places? (cc: @ashkan18 since you love naming variables)

Reply:
haha! Sorry missed this! as suggester of `commerce_verified` i have to say i don't like it either but i didn't have any better suggestion. `verified_seller` sounds better to me though! let me know if I can help in updating name elsewhere.

Reply:
renamed in https://github.com/artsy/metaphysics/pull/1983/commits/fb4a117cba5c779b275e0ff09209a6adcddc784c. Also added #dontmerge since it now depends on gravity PR: https://github.com/artsy/gravity/pull/12615